mbof/models.py

```diff
@@ -120,8 +117,8 @@ def maybe(self):
         )
         return guestTotal
 
-    def __str__(self):
-        return str(self.eventText) + ' (' + self.__class__.__name__ + ': ' + str(self.id) + ')'
+    def __unicode__(self):
+        return '%s %s %s' % (self.eventText, self.__class__.__name__, self.id)
```

Comment:
Using `format()` for substitution:

```python
'{self.eventText} {self.__class__.__name__} {self.id}'.format(**locals())
```

It's longer than the %-substitution version, but it's more readable.  Some IDEs support symbol suggestion/completion within strings when using format.